components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -490,11 +481,16 @@ public DataSection genesBody(Patient patient) throws Exception
                     continue;
                 }
                 String value = gene.get(field);
-                if (!"comments".equals(field)) {
-                    cell = new DataCell(valueTranslates.get(value), x++, y);
+                if (field.equals("strategy")) {
+                    value = parseMultivalueField(value, strategyTranslates, "PhenoTips.GeneClass_strategy_");
+                } else if (field.equals("status")) {
+                    value = parseMultivalueField(value, strategyTranslates, "PhenoTips.GeneClass_status_");
                 } else {
-                    cell = new DataCell(value, x++, y);
+                    value =
```

Comment:
Although currently the options are only those 3, it's better to check explicitly if (field.equals("comments")) so that if at any point there will be a new value, it could be added to the list without changes to the if. Even better, switch(field) which is supported from Java 7.

(Generally, it's better to check if ("comments".equals(field)) to prevent a potential NPE).